models/aave/ethereum/aave_v2_ethereum_interest_rates.sql

```diff
@@ -0,0 +1,13 @@
+{{ config(materialized='view', alias='aave_interest') }}

```

Comment:
final piece of feedback: with this value for alias, the usage of 'aave' is in both schema name and view name output:
aave_ethereum.aave_interest 

is that good with you or prefer a cleaner view name knowing 'aave' is in the schema?
@chuxinh @SohamP99

Reply:
yea agreed, i think having the interest by itself should be good here

Reply:
Alright, I have edited the alias to have only interest